app/javascript/packages/verify-flow/steps/password-confirm/password-confirm-step.tsx

```diff
@@ -1,12 +1,28 @@
 import type { ChangeEvent } from 'react';
 import { t } from '@18f/identity-i18n';
-import { FormStepsButton } from '@18f/identity-form-steps';
+import { Button } from '@18f/identity-components';
+import { FormStepsButton, useHistoryParam } from '@18f/identity-form-steps';
 import type { FormStepComponentProps } from '@18f/identity-form-steps';
+import { ForgotPassword } from './forgot-password';
 import type { VerifyFlowValues } from '../../verify-flow';
 
 interface PasswordConfirmStepStepProps extends FormStepComponentProps<VerifyFlowValues> {}
 
 function PasswordConfirmStep({ registerField, onChange }: PasswordConfirmStepStepProps) {
+  const [path, setPath] = useHistoryParam(undefined, { basePath: '/verify/v2/password_confirm' });
```

Comment:
Per prior point, we should see if we can add this path to a context, so we don't have to hard-code it.

```tsx
<VerifyFlowContext value={{ basePath, resetPasswordURL }}>
```